contents/docs/experiments/common-questions.mdx

```diff
@@ -50,3 +50,12 @@ Once you create the experiment, go to the feature flag, scroll down to "Release
 import FAQFalseOrNoneEvents from "../feature-flags/snippets/faq-false-or-none-events.mdx"
 
 <FAQFalseOrNoneEvents />
+
+## Why are the numbers in my A/B test lower than when I create the same insight directly?
+
+This can be a common problem when you're running an A/B test with the metric being a pageview: If this is the first pageview for a user, feature flag values might not load in time, which means that people don't yet see the experiment. We remove these people from the experiment calculation. To confirm this is the case, you can add a breakdown by the A/B test feature flag to your custom insight, and check the flag value is "None" for everyone removed from the experiment.
```

Comment:
@Lior539 @ivanagas - can you make this clearer somehow? Juraj's attempt below to give you a better idea of what we want

Reply:
_Experiment results only count events that include the experiment’s feature flag. Sometimes, when experiment events are sent, the flags might not be loaded yet, so these events won't have the flag information. Once the flags are loaded, all new events will include this information. This means that some experiment events might have the flag, while others do not.

While the experiment results will only show events with the flag, in an Insight, you might see all events counted, regardless of whether they have the flag information or not. To confirm this is the case, you can add a breakdown by the A/B test feature flag to your custom insight, and check the flag value is "None" for everyone removed from the experiment._